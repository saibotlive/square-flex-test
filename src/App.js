import React, { Component, Fragment } from 'react';
import { GlobalStyle } from './App.styles';
import Content from './components/Content';

class App extends Component {
  render() {
    return (
      <Fragment>
        <GlobalStyle />
        <Content />
      </Fragment>
    );
  }
}

export default App;
