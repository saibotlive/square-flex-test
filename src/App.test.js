import React from 'react';
import { render, cleanup } from 'react-testing-library';
import App from './App';

describe('<Content/>', () => {
  afterEach(cleanup);

  it('should render content', async () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
