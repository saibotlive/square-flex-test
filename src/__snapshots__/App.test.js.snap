// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content/> should render content 1`] = `
<div>
  <div
    class="sc-bdVaJa hOuLKQ"
  />
</div>
`;
